pythran/tests/openmp.legacy/omp_parallel_num_threads.py

```diff
@@ -1,3 +1,5 @@
+import unittest
+
 unittest.skip
```

Comment:
As this is not used as a decorator, this actually has no effect... I'm worried by the fact that this never triggered a failure, need to check that :-/